util/docker_build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ $# -gt 2 ]; then
+    echo "Usage: $0 [KEYBOARD [KEYMAP]]"
+    exit 1
+elif [ ! "$(command -v docker)" ]; then
+    echo "Error: docker not found"
+    echo "See https://docs.docker.com/install/#supported-platforms for installation instructions"
+    exit 2
+fi
+
+path=$(pwd -W 2>/dev/null) || path=$PWD  # Use Windows path if on Windows
+
+docker run --rm \
+    -e keyboard=${1-$(read -p "keyboard=" kb; echo $kb)} \
```

Comment:
I'd throw a `-r` in the `read` argument so it backslashes don't get messed up. 

```suggestion
-e keyboard=${1-$(read -rp "keyboard=" kb; echo $kb)}
```

Reply:
Good catch!

Reply:
@mechmerlin Could you mark this as resolved or "unrequest" the changes?